tests/e2e/utils/flows.js

```diff
@@ -589,46 +588,30 @@ export const merchantWCP = {
 		} );
 	},
 
-	openDisputeDetails: async ( disputeDetailsLink ) => {
-		await Promise.all( [
-			page.goto( WC_ADMIN_BASE_URL + disputeDetailsLink, {
-				waitUntil: 'networkidle0',
-			} ),
-			uiLoaded(),
-		] );
-		await uiLoaded();
-	},
-
 	openChallengeDispute: async () => {
-		await Promise.all( [
-			evalAndClick(
-				// eslint-disable-next-line max-len
-				'div.transaction-details-dispute-details-body div.transaction-details-dispute-details-body__actions button.components-button.is-primary'
-			),
-			page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-			uiLoaded(),
-		] );
+		// Click the challenge dispute button.
+		await evalAndClick( '[data-testid="challenge-dispute-button"]' );
+		await page.waitForNavigation( { waitUntil: 'networkidle0' } );
+		await uiLoaded();
 	},
 
 	openAcceptDispute: async () => {
-		await Promise.all( [
```

Comment:
I'm unsure why these were to be concurrently with `Promise.all`, but for the updated actions I've opted to run them sequentially.